src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <p>
+        {' Posted by  '}
+        <UserInfo
+          user={post.user}
+        />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+
+    <hr />
+
+    {Object.keys(post.comments).length > 0
```

Comment:
```suggestion
    {!!post.comments.length
```

Reply:
<img width="621" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/70220667/2c870a07-72cc-4685-8fb6-2828ffe7b2fe">